packages/contracts/tasks/runner/benchmarks.ts

```diff
@@ -12,14 +12,18 @@ task("benchmark", "Run benchmarks").setAction(async (_, hre) => {
 
   const deployer = await deployment.getDeployer();
 
+  const isIncremental = hre.network.name !== "hardhat" && hre.network.name !== "localhost";
```

Comment:
Why is `isIncremental` based on whether the network is a local network?

Asking as I believe it is a flag when deploying contracts

Reply:
It's because it doesn't make sense to redeploy for testnets and mainnets. For local node, there is a infinite amount of ether so no need to worry about it. Plus, local node sometimes has issues with incremental mode so that's why the contracts are redeployed.

Reply:
ah yes understood, thanks